UsersController.cs

```diff
@@ -0,0 +1,9 @@
+[HttpPost("delete/{id}")]
+public void Delete(uint id)
```

Comment:
We should use async model and return `Task<ActionResult>` and add cancellation token.

```suggestion
public async Task<ActionResult> Delete(uint id, CancellationToken cancellationToken)
```